packages/forms-web-app/__tests__/unit/controllers/projects/documents/filters/filter-params.test.js

```diff
@@ -0,0 +1,54 @@
+const {
+	filterParams
+} = require('../../../../../../src/controllers/projects/documents/utils/filters/filter-params');
+describe('#filterParams', () => {
+	describe('When getting the filter params', () => {
+		describe('and the query has duplicate keys', () => {
+			const params = {
+				foo: ['1', '2', '3'],
```

Comment:
use of foo bar.

Should we switch to filter-group-1-item-1 similar to the refactor in the other test